src/bijectors/ordered.jl

```diff
@@ -0,0 +1,78 @@
+"""
+    OrderedBijector()
+
+A bijector mapping ordered vectors in ℝᵈ to an unordered vectors in ℝᵈ.
+
+## See also
+- [Stan's documentation](https://mc-stan.org/docs/2_27/reference-manual/ordered-vector.html)
+  - Note that this transformation and its inverse are the _opposite_ of in this reference.
+"""
+struct OrderedBijector <: Bijector{1} end
+
+"""
+    ordered(d::Distribution)
+
+Returns a `Distribution` whose domain is now ordered vectors.
+"""
+ordered(d::Distribution) = Bijectors.transformed(d, OrderedBijector())
```

Comment:
Should this be restricted to
```suggestion
ordered(d::ContinuousMultivariateDistribution) = Bijectors.transformed(d, OrderedBijector())
```
?

Reply:
Yes, that's a good point :+1: Currently it would work for any `MultivariateDistribution` (`transformed` will fail for non-multivariate due to the dimensionality-thingy).